3-10-nl.srt

```diff
@@ -0,0 +1,149 @@
+0:00:00.480,0:00:05.220
+Kijk, Karel kan ons nu gedag zeggen maar het
+is nog wat leuker als hij een vraag
+
+0:00:05.220,0:00:07.800
+ kan stellen over jouw favoriete onderwerp.
+
+0:00:07.810,0:00:10.639
+Nou, mijn favoriete onderwerp is toevallig schildpadden.
+
+0:00:10.639,0:00:14.289
+Ik ben gek op schildpadden, ik heb twee schildpadden
+als huisdier.
+
+0:00:14.289,0:00:16.690
+Dus ik ga mijn quiz over schildpadden doen.
```

Comment:
Ik realiseer me nu dat we kwis en quiz volgens mij een beetje door elkaar heen gebruiken. Ik heb geen voorkeur, maar misschien moeten we dat even "replace all"'en naar één van beide.